### PR TITLE
Bye internet explorer

### PR DESCRIPTION
**Pull request by gesa**
### Summary

Updated branch for removing _all_ the snowflake code for Internet Explorer 

### Checklist

- [x] Added a changelog entry

- @gesa, @crookedneighbor

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,9 +20,10 @@ env:
   # - TEST="test:hint"
   - TEST="test:publishing"
   - TEST="test:integration:paypal-only" ONLY_BROWSERS=chrome,safari,firefox IS_INTEGRATION_TEST=true
-  - TEST="test:integration:paypal-only" ONLY_BROWSERS=ie IS_INTEGRATION_TEST=true
   - TEST="test:integration:paypal-skipped" ONLY_BROWSERS=chrome,firefox,safari IS_INTEGRATION_TEST=true
-  - TEST="test:integration:paypal-skipped" ONLY_BROWSERS=ie IS_INTEGRATION_TEST=true
+#  TODO: no more testing against IE and edge is evidentally still flaky
```
Comment (hollabaq86) [inline .travis.yml:24]: Do we want to address these todos in this PR or a future PR?

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ CHANGELOG
 * PayPal can now override global autovaulting preferences with `paypal.vault.autoVault`
 
 _Breaking Changes_
-* Drop support for IE 9-10
+* Drop support for Internet Explorer
```
Comment (hollabaq86) [inline CHANGELOG.md:9]: byedance

**Review by hollabaq86 — APPROVED** [review]
🎉
